api/swagger-spec/v1.json

```diff
@@ -11540,6 +11540,14 @@
      "default": {
       "type": "any",
       "description": "default values on this kind by resource name if omitted"
+     },
+     "defaultRequest": {
+      "type": "any",
+      "description": "default resource requests values on this kind by resource name if omitted"
+     },
+     "limitRequestRatio": {
+      "type": "any",
+      "description": "the ratio of limit over request that is the maximum allowed burst for the named resource. if specified, the named resource must have a request and limit that are both non-zero where limit divided by request is less than or equal to the enumerated value"
```

Comment:
I would switch the order of these two sentences, as the second one is more important than the first one. In fact once you switch them, I would rewrite the new second one to something like "This represents the max burst for the named resource."

Also, I would suggest renaming to maxLimitRequestRatio to make this more self-describing.